internal/quaiapi/quai_api.go

```diff
@@ -962,7 +958,7 @@ func (s *PublicBlockChainQuaiAPI) EstimateFeeForQi(ctx context.Context, args Tra
 	}
 
 	// Calculate the base fee
-	minGasPrice := s.b.GetMinGasPrice()
+	minGasPrice := new(big.Int).Add(s.b.CurrentBlock().BaseFee(), big.NewInt(params.GWei))
```

Comment:
Should probably increase by 20% here too

Reply:
oh yeah, i agree, will do